ops/model.py

```diff
@@ -1106,6 +1106,10 @@ def autostart(self):
         """Autostart all services marked as startup: enabled."""
         self._pebble.autostart_services()
 
+    def replan(self):
+        """Replan all services. Restart changed services, start startup services."""
```

Comment:
I think the wording in the docstring could be improved. In particular, it's unclear that the second sentence is defining "replan" (perhaps use a colon), and "startup services" is a bit vague. Perhaps:

```
"""Replan all services: restart changed services and start startup-enabled services."""
```